clientv3/client.go

```diff
@@ -291,8 +275,9 @@ func (c *Client) dialSetupOpts(target string, dopts ...grpc.DialOption) (opts []
 }
 
 // Dial connects to a single endpoint using the client's config.
-func (c *Client) Dial(endpoint string) (*grpc.ClientConn, error) {
-	return c.dial(endpoint)
+func (c *Client) Dial(ep string) (*grpc.ClientConn, error) {
+	creds := c.directDialCreds(ep)
+	return c.dial(fmt.Sprintf("passthrough:///%s", ep), creds)
```

Comment:
Can we add a short comment on `passthrough:///`?

Reply:
Sounds good, I'll add one now.

Reply:
Certainly, I've clarified why we use the passthrough resolver here.